cockatrice/src/client/ui/picture_loader.cpp

```diff
@@ -45,10 +45,11 @@ PictureToLoad::PictureToLoad(CardInfoPtr _card)
         std::sort(sortedSets.begin(), sortedSets.end(), SetDownloadPriorityComparator());
         // If the pixmapCacheKey corresponds to a specific set, we have to try to load it first.
         for (const auto &set : card->getSets()) {
-            if (QLatin1String("card_") + QString(set.getProperty("uuid")) == card->getPixmapCacheKey()) {
+            if (QLatin1String("card_") + card->getName() + QString("_") + QString(set.getProperty("uuid")) ==
```

Comment:
Still UUID?

Reply:
That's from the Json/XML/Sets in general, I didn't consider that in scope for this, since the change is already huge.